porting/finalize/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _Finalize:
+
+Finalizing the port
+===================
```

Comment:
I'm a little confused. Should this part be three steps:
1. recovery
2. system partition
3. installer
The text only really emphasizes 2 and 3, while the list only contains 1 and 3.